api_server/celery_app.py

```diff
@@ -0,0 +1,11 @@
+import os
+
+from celery import Celery
+
+app = Celery(
+    "retraining",
+    backend=os.environ.get('RESULT_BACKEND'),
+    broker=os.environ.get('BROKER_URL'),
+)
+
+app.autodiscover_tasks()
```

Comment:
Will we have celery jobs? If so, will they be stored in project root? as celery_jobs.py? Maybe we need to create a module for that stuff? Could a module solve that? (not named celery\__init__.py obviously)